src/internet_identity/src/openid.rs

```diff
@@ -157,6 +167,9 @@ fn calculate_delegation_seed(client_id: &str, (iss, sub): &OpenIdCredentialKey)
 
     blob.push(sub.len() as u8);
     blob.extend(sub.bytes());
+
+    blob.push(anchor_number as u8);
```

Comment:
This seems invalid, you should `push` the length of the anchor number in bytes before you `extend` it with the actual value:
```suggestion
    blob.push(anchor_number.to_le_bytes().len() as u8);
    blob.extend(anchor_number.to_le_bytes());
```

Reply:
Done ✅